semcore/pills/src/style/pills.shadow.css

```diff
@@ -6,6 +6,11 @@ SPills {
   outline: none;
 }
 
+SPills:focus {
```

Comment:
Because we can't use keyboardFocusedEnhance with conditions. It will always added or not to the element. In our case, we don't need to have this enhance on Pills

Reply:
We can with `use:tabIndex` expression

Reply:
I've just ensured locally that adding:
```diff
+ SPills[keyboardFocused]:focus {
```

```diff
+   static enhance = [a11yEnhance(optionsA11yEnhance), keyboardFocusEnhance()];
```

```diff
+   use:tabIndex={value !== null ? -1 : 0}
```

makes everything to work fine

Reply:
added keyboardfocus